app/controllers/message_handler_controller.rb

```diff
@@ -0,0 +1,5 @@
+class MessageHandlerController < ApplicationController
+  def handle
+
```

Comment:
OK, so this PR is just the functionality in the AKCreator classes as they're spec'ed?